homeassistant/components/http/forwarded.py

```diff
@@ -73,15 +75,31 @@ async def forwarded_middleware(
             # No forwarding headers, continue as normal
             return await handler(request)
 
-        # Ensure the IP of the connected peer is trusted
+        # Get connected IP
         assert request.transport is not None
         connected_ip = ip_address(request.transport.get_extra_info("peername")[0])
```

Comment:
I've seen stack traces from ingress where `get_extra_info` contained no `peername`. I still don't know how that happens but I think that we should guard for it.

Reply:
Added guard